change/@fluentui-react-charting-cd6618b3-30a0-474b-b2e9-9f008ff45c69.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26861&commitId=cf0e310ca73381e9d5b3232bd81acfccd94bf21c&env=prod)

✅ There was 1 screenshots added, 1 screenshots removed, 1856 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.orientation.chromium.png|0| Added|
|Card Converged.orientation.normal.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv9"/>